### PR TITLE
Refactor image replication test cases

### PR DESCRIPTION
**Pull request by yec-akamai**
## 📝 Description

Remove the hardcoded region and instead using core region with Object Storage capability for image replication. 

## ✔️ How to Test

```
make TEST_ARGS="-vvv image_basic" test
```

### CODE REVIEW

File: plugins/module_utils/linode_common.py [hunk 1]
```diff
@@ -104,7 +104,7 @@
 RETRY_STATUSES = {408, 429, 502}
 
 
-class LinodeModuleBase:
```
Comment (zliang-akamai) [inline plugins/module_utils/linode_common.py:107]: Did pylint complain about this?

**Review by zliang-akamai — APPROVED** [review]
LGTM!

File: plugins/module_utils/linode_common.py (hunk 1 above)
Comment (ykim-1) [reply plugins/module_utils/linode_common.py:107]: I also included the lint fix in Makefile (#592)

**Review by ykim-1 — APPROVED** [review]
LGTM, verified locally. Thanks for fixing the tests!

**Pull request merged**